### PR TITLE
(🥇) Use plural for zero

### PR DESCRIPTION
**Pull request by Zeckie**
resolves: https://github.com/KotlinIsland/basedmypy/issues/386 (`0 new errors` instead of `0 new error`)

Fake Tasks:
- Add reference to [Among Us](https://en.wikipedia.org/wiki/Among_Us), a 2018 [online multiplayer](https://en.wikipedia.org/wiki/Online_multiplayer) [social deduction game](https://en.wikipedia.org/wiki/Social_deduction_game).

### CODE REVIEW

**Comment by Zeckie** [comment]
@KotlinIsland I think this is ready - can you please approve workflows / review this PR.

**Comment by KotlinIsland** [comment]
@Zeckie Could you please update the description to match the template, it looks like it's missing an amongus reference.

File: test-data/unit/cmdline-based-baseline.test [hunk 1]
```diff
@@ -98,6 +98,19 @@ No errors, baseline file removed
...
+a
+[file .mypy/baseline.json]
+{
+"__baseline_metadata__": {"targets": ["file:pkg"]},
+"pkg/a.py": [{"code": "name-defined", "column": 0, "line": 1, "message": "Name \"a\" is not defined", "target": "a"}]
+}
+[out]
+Found 1 error (0 new errors) in 1 file (checked 1 source file)
+Baseline successfully written to .mypy/baseline.json
+== Return code: 0
```
Comment (KotlinIsland) [inline test-data/unit/cmdline-based-baseline.test:113]: Could you make this case separated with double newlines

Comment (Zeckie) [reply test-data/unit/cmdline-based-baseline.test:118]: or is it that some test cases are meant to have no newline?

Comment (KotlinIsland) [reply test-data/unit/cmdline-based-baseline.test:118]: Thanks, I only meant that test, but all is a bonus.

**Comment by KotlinIsland** [comment]
Thanks for the contribution.

**Pull request merged**